### ISSUE TITLE
same:attribute rule not respecting label aliases

### ISSUE THREAD

**Issue by dcworldwide** [mikeerickson/validatorjs#138]
Issue with same:attribute rule not respecting label aliases.

Code

```
        let validator = new Validator(
            {
                aEmail: this.aEmail,
                aEmail2: this.aEmail2,
            }, {
                aEmail: 'required|email',
                aEmail2: 'same:aEmail',
            }
        )

        validator.setAttributeNames({
            aEmail: 'email',
            aEmail2: 'repeated email',
        })

```

Error: 

<img width="345" alt="screen shot 2016-10-30 at 15 02 20" src="https://cloud.githubusercontent.com/assets/4898751/19834611/ed0fbaa6-9eb1-11e6-8a82-95a0ff7716c8.png">

Label should not say aEmail.

**Comment by zulis** [comment]
Have problems with 'same' rule as well :/

**Comment by robclancy** [comment]
30 October 2016 and this is still a thing huh.

**Comment by iamdtang** [comment]
@robclancy would love some help if you are interested in contributing.

**Comment by robclancy** [comment]
@skaterdav85 it legit took me the amount of time since I posted that to find how to do this. And then I see this PR https://github.com/skaterdav85/validatorjs/pull/225 which does it for the other fields and you literally just need to do a copy paste and change some text to `same`.

**Comment by robclancy** [comment]
If you can merge that PR I can make a new PR to do the copy pasting that is needed.

**Comment by iamdtang** [comment]
I just replied on that PR asking the author to add some tests since I'm not clear what those changes were for. It has been awhile since I've worked on this library and the last thing I want to do is start breaking things.

**Comment by iamdtang** [comment]
@robclancy released on 3.13.6
